sci-electronics/librepcb/librepcb-0.1.0_rc2.ebuild

```diff
@@ -7,15 +7,16 @@ inherit qmake-utils
 
 DESCRIPTION="Free EDA software to develop printed circuit boards"
 HOMEPAGE="http://librepcb.org/"
```

Comment:
You can use `https` there.

Reply:
please look https://github.com/gentoo/gentoo/pull/10896, it fresher